sdk/search/Microsoft.Azure.Search/tests/Tests/IndexerTests.cs

```diff
@@ -334,13 +334,22 @@ public void CanRunIndexerAndGetIndexerStatus()
 
                 Assert.Equal("1", oldestResult.Errors[0].Key);
                 Assert.Equal("Key field contains unsafe characters", oldestResult.Errors[0].ErrorMessage);
+                Assert.Equal("errorSourceName", oldestResult.Errors[0].Name);
```

Comment:
Why do the expected values look like property names? This is confusing. Where are these values coming from?

Reply:
This is what we mocked from the API - I'll update the mock to be better and then in a later commit fix this